test-suite-geb/build.gradle

```diff
@@ -0,0 +1,50 @@
+plugins {
+    id "io.micronaut.build.internal.convention-geb-base"
+}
+
+tasks.named("test") {
+    systemProperty "geb.env", System.getProperty('geb.env')
+    systemProperty "webdriver.chrome.driver", System.getProperty('webdriver.chrome.driver')
+    systemProperty "webdriver.gecko.driver", System.getProperty('webdriver.gecko.driver')
+}
+
+if (System.getProperty('geb.env')) {
+    apply plugin:"com.energizedwork.webdriver-binaries"
+
+    webdriverBinaries {
+        chromedriver "${chromedriverVersion}"
+        geckodriver "${geckodriverVersion}"
+    }
+}
+
+micronautBuild {
+    core {
+        usesMicronautTestJunit()
+        usesMicronautTestSpock()
+    }
+}
+
+dependencies {
+    testImplementation "cglib:cglib-nodep:3.3.0"
+    testImplementation "org.objenesis:objenesis:3.1"
+    testImplementation libs.managed.netty.codec.http
+    testImplementation(project(":inject-groovy")) {
+        exclude module: 'groovy'
+    }
+
+    testImplementation libs.geb.spock
+    testImplementation libs.spock.for.geb
+    testImplementation "org.codehaus.groovy:groovy-test:3.0.12"
```

Comment:
Is this intentional that it uses an old version?

Reply:
Yes, `test-suite-geb` runs Geb on Groovy 3 (as there's no Geb that works with Groovy 4 -- [yet](https://github.com/geb/issues/issues/668))